.github/workflows/project_tests.yml

```diff
@@ -32,6 +32,9 @@ jobs:
           - engine: dataflow
             sanitizer: dataflow
             architecture: x86_64
+          - engine: libfuzzer
```

Comment:
Can you add a comment on the reason we are only enabling undefined sanitizer with aarch64 for now.

Reply:
Is this enabled for all project languages by default ?